src/App.tsx

```diff
@@ -28,6 +29,12 @@ const ScrollToTop = () => {
 const App = () => {
   useEffect(() => {
     displayCareConsoleArt();
+    if (
+      import.meta.env.REACT_DISABLE_PATIENT_LOGIN === "true" &&
+      window.location.pathname === "/"
+    ) {
+      navigate("/login", { replace: true });
+    }
```

Comment:
instead of doing this, i'd recommend changing the router to conditionally do a redirect or show the login selector page instead... i don't see a reason for doing this in a useEffect in the first place.